soundcraft/constants.py

```diff
@@ -0,0 +1,69 @@
+#
+# Copyright (c) 2020 Jim Ramsay <i.am@jimramsay.com>
+# Copyright (c) 2020,2021 Hans Ulrich Niedermann <hun@n-dimensional.de>
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+#
+########################################################################
+#
+# This file is intended to work without importing any non-standard
+# Python libraries; ideally without any imports at all.
+#
+# This allows us to import this both from the setuptools `setup.py`
+# and from any code inside `soundcraft/` as well.
+#
+########################################################################
+
+# Package name
+PACKAGE = "soundcraft-utils"
+VERSION = "0.4.90"
+
+# Executable names
+BASE_EXE_CLI = "soundcraft_ctl"
+BASE_EXE_GUI = "soundcraft_gui"
+BASE_EXE_SERVICE = "soundcraft_session_service"
+BASE_EXE_INSTALLTOOL = "soundcraft_installtool"
+
+
+# Gtk Application ID of the soundcraft-utils GUI
+# https://developer.gnome.org/gio/stable/GApplication.html#g-application-id-is-valid
+APPLICATION_ID = "io.github.soundcraft_utils.gui"
+
+
+# The complete set of D-Bus API strings
+BUSNAME = "io.github.soundcraft_utils.notepad"
+DEVICE_INTERFACE = f"{BUSNAME}.device"
+SERVICE_INTERFACE = BUSNAME
+MGRPATH = "/io/github/soundcraft_utils/notepad"
+
+
+# Helpful for removing remnants of soundcraft-utils 0.4.0 and earlier
+OLD_BASE_EXE_SERVICE = "soundcraft_dbus_service"
+OLD_STATEDIR = "/var/lib/soundcraft-utils"
```

Comment:
Open question: Since we're changing the name, maybe it's okay to just say "No upgrade; Just delete the old and install the new" - Not like there's a huge amount of data in the old statedir, anyway :)

Reply:
I was too lazy to remove all that compat stuff in the last few days, because I had already written and tested it last year.

I think we could or should remove it after the big rename, though.

Reply:
Cool.